spec/models/recipe_spec.rb

```diff
@@ -12,4 +17,10 @@
     it {should have_many :recipe_ingredients}
     it {should have_many(:ingredients).through(:recipe_ingredients)}
   end
+
+  describe '#total_ingredient_cost' do
+    it 'returns the total cost of all ingredients' do
+      expect(@recipe_1.total_ingredient_cost).to eq(90)
+    end
+  end
```

Comment:
```ruby
def total_cost
  90
end
```

Does your assertion disprove that this is what your method looks like? Probably not. My suggestion, for instance methods, is to test more than 1 instance object so that way your code proves that the value isn't hard-coded and is dynamic.

```ruby
expect(@recipe_1.total_cost).to eq(90)
expect(@recipe_2.total_cost).to eq(12) # create another recipe
```